great_expectations/compatibility/aws.py

```diff
@@ -25,3 +25,13 @@
     from botocore import exceptions
 except ImportError:
     exceptions = AWS_NOT_IMPORTED
+
+try:
+    import sqlalchemy_redshift
+except ImportError:
+    sqlalchemy_redshift = AWS_NOT_IMPORTED
```

Comment:
I think within the same file is fine. Perhaps a different error message would be better as to not tell user to install something they may not require
```suggestion
    sqlalchemy_redshift =  NotImported(
    "Redshift connection components are not installed, please 'pip install sqlalchemy_redshift'"
)
```

Reply:
@TrangPham Different message created -- thanks!